### DISCUSSION TITLE
How do I get the text view of an article by default?

### DISCUSSION THREAD

**Discussion by UtkarshVerma** [spacecowboy/Feeder#416] [Q&A] [answered]
My university posts its announcements via an RSS feed. I added the feed to Feedr and it is able to fetch the articles, but every time I get this:

<img src="https://github.com/user-attachments/assets/a0d7754c-7731-442d-bef9-0b094e90a5d1" width="250">

I have to press the "Fetch full article button" every time to get to the actual announcement. Is there a way I can make this the default behaviour?

<img src="https://github.com/user-attachments/assets/bc9dff79-9956-4191-b57c-979ae1e498e7" width="250">

**Comment by spacecowboy** [comment]
Edit the feed and select full text ny default

**Reply by UtkarshVerma** [reply]
I had it set already, still the same behaviour.

**Reply by spacecowboy** [reply, marked as answer]
The first image is the full text. So for this feed you should *disable full text by default* since the site is blocking you from reading it without cookies.
